libs/designsystem/modal/src/modal/footer/modal-footer.component.scss

```diff
@@ -43,6 +43,7 @@ ion-footer {
 :host(.inline) ion-footer {
   background: transparent;
   box-shadow: none;
+  @include themes.apply-light-theme;
```

Comment:
Just a FYI-comment 😊:

A common best practice using Sass is to apply mixins before "local" declarations. It can make it easier to spot if some properties are overwritten. 

However, in this case it doesn't matter, since `themes.apply-light-theme` doesn't apply declarations for `background` or `box-shadow`

Reply:
Makes great sense ! I have rearranged